DOCUMENTATION.md

```diff
@@ -363,25 +363,33 @@ Returns an object with the following keys:
 Indexes of each string table have corresponding constants
 
 * `PI_MODEL_NAME`
-  * `PI_MODEL_UNKNOWN`
-  <span class="api-info-list"><code> >= 2.0.0 </code></span>
   * `PI_MODEL_A`
   <span class="api-info-list"><code> >= 1.1.0 </code></span>
-  * `PI_MODEL_AP`
-  <span class="api-info-list"><code> >= 2.1.0 </code></span>
   * `PI_MODEL_B`
   <span class="api-info-list"><code> >= 1.1.0 </code></span>
+  * `PI_MODEL_AP`
+  <span class="api-info-list"><code> >= 2.1.0 </code></span>
   * `PI_MODEL_BP`
   <span class="api-info-list"><code> >= 2.0.0 </code></span>
+  * `PI_MODEL_2`
+  <span class="api-info-list"><code> >= 2.1.0 </code></span>
+  * `PI_ALPHA`
+  <span class="api-info-list"><code> >= 2.1.0 </code></span>
   * `PI_MODEL_CM`
   <span class="api-info-list"><code> >= 1.1.1 </code></span>
-  * `PI_MODEL_2`
+  * `PI_MODEL_07`
+  <span class="api-info-list"><code> >= 2.1.0 </code></span>
+  * `PI_MODEL_3`
+  <span class="api-info-list"><code> >= 2.1.0 </code></span>
+  * `PI_MODEL_ZERO`
```

Comment:
Since new constants are added the version should be bumped, add the documentation should reflect it accordingly.

Also since some are renamed/removed, it's a breaking change, i suggest `2.2.0` then